quesma/single_table/const.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package single_table
+
+import (
+	"database/sql"
+	"quesma/logger"
+)
+
+const TableName = "quesma_all_logs"
```

Comment:
```suggestion
const TableName = "quesma_common_table"
```
how about this?

Reply:
Looks good. I'll refactor it. This feature is called `Common Table` now.